src/stdlib/native/int.no

```diff
@@ -18,10 +19,90 @@ fn Uint8.new(val: Field) -> Uint8 {
     };
 }
 
+// u16
+struct Uint16 {
+    inner: Field,
+    bit_len: Field,
+}
+
+fn Uint16.new(val: Field) -> Uint16 {
+    let bit_len = 16;
+
+    // range check
+    let ignore_ = bits::to_bits(bit_len, val);
+
+    return Uint16 {
+        inner: val,
+        bit_len: bit_len
+    };
+}
+
+// u32
+struct Uint32 {
+    inner: Field,
+    bit_len: Field,
+}
+
+fn Uint32.new(val: Field) -> Uint32 {
+    let bit_len = 32;
+
+    // range check
+    let ignore_ = bits::to_bits(bit_len, val);
```

Comment:
usually the syntax is `let _ =` or `let _bits = `

Reply:
the current rule doesn't allow starting with `_`